src/UniqueFileIterator.php

```diff
@@ -28,10 +28,6 @@ public function accept()
         /** @var SplFileInfo $file */
         $file = $this->current();
 
-        if ($file->isDir() || $file->isLink()) {
-            return false;
-        }
-
         $path = $file->getRealPath();
 
         if (array_key_exists($path, $this->visitedElements)) {
```

Comment:
isset is actually faster on php 5 than array_key_exists - do you want me to change this @keradus?